spec/features/agents/agent_can_search_plage_ouverture_spec.rb

```diff
@@ -1,23 +1,60 @@
 # frozen_string_literal: true
 
 describe "Agent can search plage ouverture" do
-  specify do
-    organisation = create(:organisation)
-    agent = create(:agent, organisations: [organisation])
-
-    perm_enfance = create(:plage_ouverture, title: "Permanence Enfance", agent: agent, organisation: organisation)
-    perm_scolaire = create(:plage_ouverture, title: "Permanence Scolaire", agent: agent, organisation: organisation)
+  let(:organisation) { create(:organisation) }
+  let(:agent) { create(:agent, organisations: [organisation]) }
+  let!(:perm_enfance) { create(:plage_ouverture, title: "Permanence Enfance en cours", agent: agent, organisation: organisation) }
+  let!(:perm_scolaire) { create(:plage_ouverture, title: "Permanence Scolaire en cours", agent: agent, organisation: organisation) }
+  let!(:expired_perm_enfance) { create(:plage_ouverture, :expired, title: "Permanence Enfance passée", agent: agent, organisation: organisation) }
+  let!(:expired_perm_scolaire) { create(:plage_ouverture, :expired, title: "Permanence Scolaire passée", agent: agent, organisation: organisation) }
 
+  before do
     login_as(agent, scope: :agent)
     visit admin_organisation_agent_plage_ouvertures_path(organisation, agent)
+  end
+
+  it "displays the correct elements before search" do
+    expect(find_link("En cours")[:class]).to include("active")
+    expect(find_link("Passées")[:class]).not_to include("active")
+    expect(page).to have_content(perm_enfance.title).once
+    expect(page).to have_content(perm_scolaire.title).once
+    expect(page).not_to have_content(expired_perm_enfance.title)
+    expect(page).not_to have_content(expired_perm_scolaire.title)
+  end
 
-    expect(page).to have_content(perm_enfance.title)
-    expect(page).to have_content(perm_scolaire.title)
+  it "displays the correct elements after search" do
+    fill_in :search, with: "sco"
+    click_button "Rechercher"
+
+    expect(page).not_to have_content(perm_enfance.title)
+    expect(page).to have_content(perm_scolaire.title).once
+    expect(page).not_to have_content(expired_perm_enfance.title)
+    expect(page).not_to have_content(expired_perm_scolaire.title)
+  end
+
+  it "preserve the search after tab changes" do
```

Comment:
petites typo : "it preserve**s** the search after tab change"

Reply:
```suggestion
  it "preserves the search after tab changes" do
```